.github/workflows/functional_test.yaml

```diff
@@ -3,6 +3,12 @@ name: Splunk Platform Functional Test
 on:
   pull_request:
 
+env:
+  GO_VERSION: 1.24.1
```

Comment:
I like pinning the go version. We should probably keep the major and minor version in sync with the [collector](https://github.com/signalfx/splunk-otel-collector/blob/a389cf7e71ec107394be888e4d9d28f12df08afd/go.mod#L3C1-L3C10)